code/datums/martial/psychotic_brawl.dm

```diff
@@ -3,42 +3,156 @@
 	id = MARTIALART_PSYCHOBRAWL
 
 /datum/martial_art/psychotic_brawling/disarm_act(mob/living/carbon/human/A, mob/living/carbon/human/D)
-	return psycho_attack(A,D)
+	return psycho_disarm(A,D)
 
 /datum/martial_art/psychotic_brawling/grab_act(mob/living/carbon/human/A, mob/living/carbon/human/D)
-	return psycho_attack(A,D)
+	return psycho_grab(A,D)
 
 /datum/martial_art/psychotic_brawling/harm_act(mob/living/carbon/human/A, mob/living/carbon/human/D)
 	return psycho_attack(A,D)
 
-/datum/martial_art/psychotic_brawling/proc/psycho_attack(mob/living/carbon/human/A, mob/living/carbon/human/D)
+/datum/martial_art/psychotic_brawling/proc/psycho_disarm(mob/living/carbon/human/A, mob/living/carbon/human/D)
 	var/atk_verb
-	switch(rand(1,8))
+	switch(rand(1,6))
 		if(1)
+			A.do_attack_animation(D, ATTACK_EFFECT_DISARM)
+			playsound(D, 'sound/weapons/slap.ogg', 50, TRUE, -1)
+			var/throwtarget = get_edge_target_turf(A, get_dir(A, get_step_away(D, A)))
+			D.throw_at(throwtarget, 4, 2, A)
+			D.visible_message("<span class='warning'>[A] shoves [D] across the room!</span>", "<span class='userdanger'>[A] shoves you with inhuman strength!</span>")
+			D.Knockdown(60)
+			atk_verb = "shoves"
+		if(2,3)
+			var/throw_dir = pick(GLOB.alldirs)
+			var/atom/throw_target = get_edge_target_turf(A, throw_dir)
+			A.throw_at(throw_target, 1, 4)
+			A.visible_message("<span class='warning'>[A] trips over [A.p_them()]self!</span>", "<span class='userdanger'>You stumble and trip!</span>")
+			A.Paralyze(40)
+			A.Knockdown(60)
+		if(4)
+			A.visible_message("<span class='warning'>[A] starts throwing a tantrum!</span>")
+			A.emote("scream")
+			atk_verb = "throws a tantrum at"
+			A.Paralyze(12)
+			A.Knockdown(25)
+			for(var/i in list(NORTH,SOUTH,EAST,WEST,EAST,SOUTH,NORTH,SOUTH,EAST,WEST,EAST,SOUTH))
```

Comment:
```suggestion
			for(var/i in GLOB.cardinals)
```

Reply:
If you want it to loop twice just put that loop in another for loop. But I don't really know why you needed multiple iterations in the first place

Reply:
Not my code, ripped it from plasmafist, could just change it to smth else

Reply:
Doesn't matter, I'd rather have one shit proc than two :pensive: